wgsl/index.bs

```diff
@@ -970,9 +970,24 @@ pipeline_stage
 ## Statements ## {#statements}
 
 <pre class='def'>
+branch_stmt
+  : RETURN logical_or_expression SEMICOLON
+      OpReturn
+      OpReturnValue
+  | break_stmt SEMICOLON
+  | continue_stmt SEMICOLON
+  | KILL SEMICOLON
+
 body_stmt:
   : BRACKET_LEFT statements BRACKET_RIGHT
+  | branch_stmt
```

Comment:
This seems to be too general as this makes it grammatically correct to say

```
fn foo() -> f32
   return 2.3;
```

Seems like we should have an `if_body_stmt` and then we need to update the if section so you can't do something like:

```
if (a)
  break;
else if (b)
  continue;
else
 kill;
```

as I don't think that should be allowed?